docs/recipes/gulp/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const gulp = require('gulp');
+const connect = require('gulp-connect');
+const lighthouse = require('../../../lighthouse-core');
```

Comment:
If the idea is that we have cut/paste examples, users of this example will write this:

```
const lighthouse = require('lighthouse');
const perfConfig = require('lighthouse/lighthouse-core/config/perf.json');
```